src/connection.js

```diff
@@ -290,6 +290,22 @@ class Connection {
 		}
 	}
 
+	/**
+	 * Normalisation of the namespace to a value that is compatible with the OpenEO specs.
```

Comment:
This has not been standardized yet, thus declaring it as experimental (as all other mention of namespaces).

```suggestion
	 * Normalisation of the namespace to a value that is compatible with the OpenEO specs - EXPERIMENTAL.
```